.github/ISSUE_TEMPLATE/request.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Feature Request
```

Comment:
Just to be consistent with the `Bug report` above.  I could go either way.  And yeah, this obviously is a nit pick... `:]`

```suggestion
name: Feature request
```